src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+.big-label {
+  position: relative;
+}
+
+.big-label::before {
+  position: absolute;
+  top: 0;
+  left: 26px;
+  content: "";
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: 19px;
+  width: 19px;
+}
+
+.big-input {
+  font-size: 16px;
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-input::placeholder {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.big-input:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.big-input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  border-radius: 3px;
+  font-weight: 900;
+}
+
+.small-input {

```

Comment:
You have similar properties for both small and big inputs. Give them a common class (e.g. "input") and move them under its selector
![image](https://user-images.githubusercontent.com/96409563/231077319-4a95798d-fd08-4105-b130-edeeadac143c.png)